src/setup/process.rs

```diff
@@ -0,0 +1,58 @@
+//! Set of convenience function to manage rippled process.
+
+use std::{
+    net::SocketAddr,
+    process::{Child, Command, Stdio},
+    time::Duration,
+};
+
+use tokio::io::AsyncWriteExt;
+
+use crate::{setup::config::NodeMetaData, wait_until};
+
+pub const CONNECTION_TIMEOUT: Duration = Duration::from_secs(2);
```

Comment:
any reason why it's not in https://github.com/runziggurat/xrpl/blob/main/src/tools/constants.rs?

Reply:
I wanted to keep related things close to each other. Moved to consts.